client/navigation/NavigationStore.js

```diff
@@ -0,0 +1,31 @@
+
+/**
+ * Module dependencies.
+ */
+
+var createStore = require('dispatchr/utils/createStore');
+
+module.exports = createStore({
+    initialize: function () {
+    	this.state = {
+            currentPage: null
+        };
+    },
+    storeName: 'NavigationStore',
+    handlers: {
+        'NAVIGATE': function (route) {
+            var store = this.dispatcher.getStore('NavigationStore');
+            store.state.currentPage = route.name;
+            store.emit('NAVIGATE');
```

Comment:
Seems unusual for event names to be ALLCAPS. Does this convention come from somewhere?

Also, don't need quotes around the NAVIGATE key above.